libs/features/georoutingv2/georoutingv2.js

```diff
@@ -100,15 +100,17 @@ async function getAvailableLocales(locales) {
 
 function getGeoroutingOverride() {
   const urlParams = new URLSearchParams(window.location.search);
-  const param = urlParams.get('hideGeorouting');
-  const hideGeorouting = param || getCookie('hideGeorouting');
-  if (param === 'on') {
+  const param = urlParams.get('georouting');
+  const georouting = param || getCookie('georouting');
+  const domain = window.location.host === 'adobe.com'
```

Comment:
Looks like you're using `domain` only if `param` is `off`, so this could be moved inside the L107 conditional

Reply:
Thanks for the review @overmyheadandbody . I have updated the code